app/assets/stylesheets/application.scss

```diff
@@ -105,3 +137,102 @@ body {
   width: 250px;
   padding: 5px;
 }
+
+// Masonary
+* {
+  -webkit-box-sizing: border-box;
+          box-sizing: border-box;
+}
+
+
+// Grid
+
+.grid {
+  background: $mediumaquamarine;
+  max-width: 2000px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+// clearfix
+.grid:after {
+  content: "";
+  display: block;
+  clear: both;
+}
+
+// Grid-items
+.trip-item {
+  background: $darkseagreen;
+  border: 2px solid #333;
+  border-color: hsla(0, 0%, 0%, 0.5);
+  float: left;
+  height: auto;
+  margin: 10px;
+  padding: 20px;
+  width: 300px;
+}
+
+.post-item h4 {
+  text-align: center;
+}
+
+.album-item {
+  float: right;
+  margin: 10px;
+  max-width: 300px;
+  max-height: 100%;
+}
+
+.album-item-2 {
+  float: left;
+  margin: 10px;
+  max-width: 400px;
+  max-height: 100%;
+}
+
+.album-item-3 {
+  float: left;
+  margin: 10px;
+  max-width: 500px;
+  max-height: 100%;
+}
+
+.album-item-4 {
+  float: left;
+  margin: 10px;
+  max-width: 200px;
+  max-height: 100%;
+}
+
+.post-item {
+  background: rgba(255, 255, 255, 0.5);
+  border: 2px solid #333;
```

Comment:
Color literals like `#333` should only be used in variable declarations; they should be referred to via variable everywhere else.